source/module_io/output_dmk.h

```diff
@@ -0,0 +1,28 @@
+#ifndef MODULE_IO_OUTPUT_DMK_H
+#define MODULE_IO_OUTPUT_DMK_H
+#include "module_basis/module_ao/parallel_orbitals.h"
+#include "module_elecstate/module_dm/density_matrix.h"
+#include "module_io/output_interface.h"
+
+namespace ModuleIO
+{
+
+template <typename TK>
+class Output_DMK : public Output_Interface
```

Comment:
The class of 'Output_Interface' needs discussion

Reply:
I agree with this comment. It is useful only when we need a single base class pointer to output varies output info. But here it seems redundant. If you think it is useless here, I will delete this derivation.